backend/gn_plugin_depobio/migrations/01eb67fd7851_add_depobio_plugin_module_permission.py

```diff
@@ -0,0 +1,72 @@
+"""Add depobio module permission
+
+Revision ID: 1e2926e0b603
+Revises: 78ba67f597ee
```

Comment:
```suggestion
Revision ID: 01eb67fd7851
```

- Mettre le bon ID pour "Revision ID" : ID de la présente migration
- Retirer la ligne `Revises: [...]` car il s'agit ici de la première migration d'une branche

Reply:
oups